component-maps.yml

```diff
@@ -82,24 +82,6 @@ git:
     docker_container: []
     release_component: true
 
-  mender-conductor:
-    docker_image:
-    - mender-conductor
-    - email-sender
-    docker_container:
-    - mender-conductor
-    - mender-email-sender
-    release_component: true
-
-  mender-conductor-enterprise:
-    docker_image:
-    - mender-conductor-enterprise
-    - org-welcome-email-preparer
-    docker_container:
-    - mender-conductor
-    - mender-org-welcome-email-preparer
-    release_component: true
```

Comment:
In `component-maps.yml`, these should not be removed, but should be set to `release_component: false`. This has to do with being able to query older versions integration. It must still be able to answer this query, for example:

```
release_tool.py --version-of mender-conductor --in-integration-version 2.0.0
```

A comment might be appropriate though (I think YML supports that).

Same for the rest of the changes in `component-maps.yml`.

Reply:
Done (without comment, I believe `release_componet: false` is self-explanatory).